launcher/launcher/main.go

```diff
@@ -155,6 +156,19 @@ func startLauncher(ctx context.Context, launchSpec spec.LaunchSpec, serialConsol
 	}
 	defer containerdClient.Close()
 
+	if launchSpec.InstallGpuDriver {
```

Comment:
This shouldn't be done at the beginning of startLauncher. At minimum it should happen before the TPM check but probably after retrieving the auth token as well.